interface/src/avatar/MyAvatar.cpp

```diff
@@ -5014,6 +5014,7 @@ float MyAvatar::getAnalogSprintSpeed() const {
 
 void MyAvatar::setAnalogPlusWalkSpeed(float value) {
     _analogPlusWalkSpeed.set(value);
+    analogPlusWalkSpeedChanged(value);
```

Comment:
You'll need to `emit` because this is a `signal`
```suggestion
    emit analogPlusWalkSpeedChanged(value);
```

Reply:
Ack